pkg/registration/helpers/constants.go

```diff
@@ -0,0 +1,6 @@
+package helpers
+
+const (
+	ClusterImportConfigSecret = "cluster-import-config"
```

Comment:
why do we define a new file here? can we just put in the importer directory if it is only for importer.